client/lib/posts/actions.js

```diff
@@ -435,69 +436,77 @@ PostActions = {
 	*
 	* @api public
 	*/
-	fetchNextPage: function() {
-		var params, id, siteID;
+	fetchNextPage: function( postListStoreId = 'default' ) {
+		var params, id, siteID, postListStore;
 
-		if ( PostListStore.isLastPage() ) {
+		postListStore = postListStoreFactory( postListStoreId );
+
+		if ( postListStore.isLastPage() ) {
 			return;
 		}
 
 		Dispatcher.handleViewAction( {
-			type: 'FETCH_NEXT_POSTS_PAGE'
+			type: 'FETCH_NEXT_POSTS_PAGE',
+			postListStoreId: postListStoreId
 		} );
 
-		id = PostListStore.getID();
-		params = PostListStore.getNextPageParams();
-		siteID = PostListStore.getSiteID();
+		id = postListStore.getID();
+		params = postListStore.getNextPageParams();
+		siteID = postListStore.getSiteID();
 
 		if ( siteID ) {
 			wpcom
 				.site( siteID )
-				.postsList( params, PostActions.receivePage.bind( null, id ) );
+				.postsList( params, PostActions.receivePage.bind( null, id, postListStoreId ) );
 		} else {
 			wpcom
 				.me()
-				.postsList( params, PostActions.receivePage.bind( null, id ) );
+				.postsList( params, PostActions.receivePage.bind( null, id, postListStoreId ) );
 		}
 	},
 
-	receivePage: function( id, error, data ) {
+	receivePage: function( id, postListStoreId, error, data ) {
 		Dispatcher.handleServerAction( {
 			type: 'RECEIVE_POSTS_PAGE',
 			id: id,
+			postListStoreId: postListStoreId,
 			error: error,
 			data: data
 		} );
 	},
 
-	fetchUpdated: function() {
-		var id, params, siteID;
+	fetchUpdated: function( postListStoreId = 'default' ) {
+		var id, params, siteID, postListStore;
+
+		postListStore = postListStoreFactory( postListStoreId );
 
 		Dispatcher.handleViewAction( {
-			type: 'FETCH_UPDATED_POSTS'
+			type: 'FETCH_UPDATED_POSTS',
+			postListStoreId: postListStoreId
 		} );
 
-		id = PostListStore.getID();
-		params = PostListStore.getUpdatesParams();
-		siteID = PostListStore.getSiteID();
+		id = postListStore.getID();
+		params = postListStore.getUpdatesParams();
+		siteID = postListStore.getSiteID();
 
 		if ( siteID ) {
 			debug( 'Fetching posts that have been updated for %s since %s %o', siteID, params.modified_after, params );
 			wpcom
 				.site( siteID )
-				.postsList( params, PostActions.receiveUpdated.bind( null, id ) );
+				.postsList( params, PostActions.receiveUpdated.bind( null, id, postListStoreId ) );
```

Comment:
Does this need to pass `postListStoreId`? I'd think any instance of the `PostListStore` would be interested in receiving updated posts.